api/test/factories.ts

```diff
@@ -268,7 +268,7 @@ export const generateLicenseEntity = (overrides: Partial<LicenseEntity>): Licens
   return {
     fullName: `some-name-${randomInt()}`,
     addressLine1: `some-address-${randomInt()}`,
-    addressCity: `some-city-${randomInt()}`,
+    foreignAddressCity: `some-city-${randomInt()}`,
```

Comment:
this is LicenseEntity, not the formation stuff that you are changing. Should not be changed, this represents something else entirely